src/components/modal/MarkerModal.tsx

```diff
@@ -23,53 +52,42 @@ const MarkerModal = ({ openModal, onClose, item, ageKey }: any) => {
         <span style={{ fontSize: '24pt' }}>{item.name}</span>
       </div>
       {item.aboveAverage ? (
-        <div className="flex flex-col justify-center items-center p-3 text-white w-[100%] my-5">
-          {/* ------------------------------------------------ */}
-
-          <div className="flex flex-col justify-center items-center p-2 bg-secondary rounded-t-lg w-2/3">
-            <div className="relative">
-              <h1 className="text-3xl relative">SUPERIOR</h1>
-            </div>
-            <div className={styles.rangeValues}>
-              {item.superior} {item.units}
-            </div>
-          </div>
-          {/* ------------------------------------------------ */}
-          <div className="flex flex-col justify-center items-center p-2 bg-blue-800 w-2/3">
-            <div className="relative">
-              <h1 className="text-3xl relative">ABOVE AVERAGE</h1>
-            </div>
-
-            <div className={styles.rangeValues}>
-              {item.aboveAverage} {item.units}
-            </div>
-          </div>
-          {/* ------------------------------------------------ */}
-          <div className="flex flex-col justify-center items-center p-2 bg-neutral rounded-b-lg w-2/3">
-            <div className="relative">
-              <h1 className="text-3xl relative">AVERAGE</h1>
-            </div>
-            <div className={styles.rangeValues}>
-              {item.average} {item.units}
-            </div>
-          </div>
-        </div>
+        <BarChart chartData={data} />
       ) : (
-        <div className="flex flex-col justify-center items-center p-2 bg-accent rounded-lg w-2/3 text-white my-5">
+        <div className="flex flex-col justify-center items-center p-2 text-neutral rounded-xl w-1/2 border-2 border-neutral my-5">
           <div className="relative">
-            <h1 className="text-3xl relative">AVERAGE</h1>
+            <h1 className="text-3xl">AVERAGE</h1>
           </div>
 
-          <div className={styles.rangeValues}>
+          <div
+            style={{
+              fontFamily: 'Verdana',
+              color: 'rgb(117, 195, 117)',
+              fontSize: '16pt',
+              fontWeight: 'bold',
+            }}
+            className="text-white"
+          >
             {item.average} {item.units}
           </div>
         </div>
       )}
+
       <Accordion item={item} />
       <div className="flex gap-4 justify-center items-center border-2 p-3 my-2 w-full">
         <p style={{ fontFamily: 'Arial' }}>VideoThumb</p>
-        <a href="https://www.youtube.com/watch?v=nVsWUtN7NXI" target="_blank">
-          <img src={`src/images/${item.videoThumb}`}></img>
+        <a href={item.videoURL} target="_blank">
+          {/* <img
+            className="border-2 border-red-300 h-[80px] w-[100px]"
+            src="shoulderTest.png"
+            alt="video-link"
+          ></img> */}
+          <Image
+            src={`/src/components/modal/images/${item.videoThumb}`}
```

Comment:
This isn't working right now because of how we are calling the image asset. To be able to access an image asset in a React component you need to import it and call the asset as a variable, not provide a path for the asset: https://stackoverflow.com/questions/43823289/how-to-import-image-svg-png-in-a-react-component

We could import all of these image assets as a variables at the top of the file, or we could move them to the `/public` folder and use the file path to access them. If you put the 3 image assets in the `/public` folder, you can change this source tag to `src={`/#{item.vidoeThumb}`}`